tests/test_ivdb.py

```diff
@@ -156,24 +157,27 @@ def test_make_db_dir():
 
 
 def test_make_db_dir_exists():
-    r0 = r1 = r2 = False
+    r = np.full(4, False)
     db = test_dir + "/test-direxists"
     if isdir(db):
         shutil.rmtree(db)
     mkdir(db)
     db_exp = test_dir + "/test-direxists_1/"
     iv = ivdb.IvDb(levels=levels, data=data, db=db)
-    with pytest.warns(None) as warn_info:
+    with warnings.catch_warnings(record=True) as w:
         iv._IvDb__make_db_dir()
-    if len(warn_info) == 1:
-        r0 = True
+        warnings.simplefilter("always")
+    if len(w) == 1:
+        r[0] = True
+        if "exists" in str(w[0].message):
+            r[1] = True
```

Comment:
non-blocking suggestion: There is maybe some argument for a little method that does this test?  Something like:
```
r[0], r[1] = checkWarning(w,"exists")
```